Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -1386,6 +1445,7 @@ extension NIOHTTP2Handler {
         public var maximumBufferedControlFrames: Int = 10000
         public var maximumSequentialContinuationFrames: Int = NIOHTTP2Handler.defaultMaximumSequentialContinuationFrames
         public var maximumRecentlyResetStreams: Int = NIOHTTP2Handler.defaultMaximumRecentlyResetFrames
+        public var maximumConnectionGlitches: Int = GlitchesMonitor.defaultMaximumGlitches
```

Comment:
It would be good, I think, to include narrative docs that explain this field and what it does.